### ISSUE TITLE
Rename key used from "cache" to something bootup specific

### ISSUE THREAD

**Issue by jwlrs** [TradeMe/bootup.js#15]
Considering that bootup may live alongside other code that wants to use localStorage, it would be a good idea to use a less generic name for your key on localStorage.setItem, in order to avoid accidental collisions.  Maybe "bootupjs.cache" or something.

Otherwise, great idea!

**Comment by djmc** [comment]
Good point! Maybe being able to specify what the key is will also allow a bit of control if even that is a collision for whatever reason.
